src/components/playback/playbackmanager.js

```diff
@@ -261,18 +261,109 @@ define(['events', 'datetime', 'appSettings', 'itemHelper', 'pluginManager', 'pla
 
     function getIntros(firstItem, apiClient, options) {
 
-        if (options.startPositionTicks || options.startIndex || options.fullscreen === false || !enableIntros(firstItem) || !userSettings.enableCinemaMode()) {
+        var unwatchedOnly = userSettings.enableCinemaTrailersUnseenOnly();
+        var trailerCount = userSettings.cinemaModeTrailerCount();
+        var enableCinema = userSettings.enableCinemaMode();
+
+        if (options.startPositionTicks || options.startIndex || options.fullscreen === false || !enableIntros(firstItem) || !enableCinema || trailerCount == 0) {
             return Promise.resolve({
                 Items: []
             });
         }
 
-        return apiClient.getIntros(firstItem.Id).then(function (result) {
+        var fetchOptions = {
+            limit: 500,
+            fields: "RemoteTrailers,LocalTrailerCount,ServerID",
+            filters: unwatchedOnly ? "IsUnPlayed" : "",
+            recursive: true,
+            includeItemTypes: "Movie"
+        };
+
+        function getRemoteTrailers(item) {
+            var remoteTrailers = item.RemoteTrailers || [];
+
+            if (!remoteTrailers.length) {
+                return [];
+            }
+
+            return remoteTrailers.map(function (t) {
+                return {
+                    Name: t.Name || (item.Name + ' Trailer'),
+                    Url: t.Url,
+                    MediaType: 'Video',
+                    Type: 'Trailer',
+                    ServerId: apiClient.serverId()
+                };
+            });
```

Comment:
This seems like a frontend solution for a backend issue to me.

Reply:
@Ullmie02 was this done to match the local trailer objects or something? Just trying to get a clear picture.